test/plugin/scenarios/netty-socketio-scenario/src/main/webapp/WEB-INF/web.xml

```diff
@@ -22,6 +22,10 @@
          version="3.1">
     <display-name>skywalking-netty-socketio-scenario</display-name>
 
+    <listener>
+        <listener-class>org.apache.skywalking.apm.testcase.netty.socketio.ContextListener</listener-class>
+    </listener>
+
```

Comment:
I think this will cause unstable. If possible, move it to health-check.

Reply:
resolved. change to the health check servlet.